framework/meta/src/generate_snippets/snippet_crate_gen.rs

```diff
@@ -69,8 +69,6 @@ path = ".."
 [dependencies.multiversx-sc-snippets]
 version = "0.39.0"
 
-[workspace]
```

Comment:
The thing is, if you generate the project for the first time and don't make it a contained workspace, you also have too add the path to the root Cargo.toml.

If the dev wants to make the project permanent it better to have it compile every time, but if he only wants to play around it's better to have it separate ....

Not sure which one is better.

Reply:
Added it back.